### PR TITLE
replaced while true with while condition

### PR DESCRIPTION
**Pull request by jaykay99992**
While true :
if (condition) break

can simply become:

While not condition:

### CODE REVIEW

File: src/flask/cli.py [hunk 1]
```diff
@@ -201,13 +201,10 @@ def prepare_import(path):
     module_name = []
 
     # move up until outside package structure (no __init__.py)
-    while True:
+    while os.path.exists(os.path.join(path, "__init__.py")):
```
Comment (eddyallard) [inline src/flask/cli.py:204]: This change may impact functionnality, considering that lines 205 and 206 will not be executed prior to the condition being evaluated. The loop behaviours goes from a do...while loop to a while loop.

**Comment by jaykay99992** [comment]
@eddyallard So do you recommend rejecting it and keep the first version or you recommend different changes ?

**Comment by eddyallard** [comment]
@jaykay9999 Considering that there are no do...while loops in python and that there are no checks associated with this pull request to validate it's functionnality, I would recommend rejecting it and keeping the first version.

**Comment by jaykay9999** [comment]
Pull request rejected.

**Pull request closed without merging**